libs/ipc/inc/public/tfc/ipc/details/impl.hpp

```diff
@@ -299,11 +305,23 @@ class slot_callback : public std::enable_shared_from_this<slot_callback<type_des
   using value_t = type_desc::value_t;
   static auto constexpr direction_v = slot<type_desc>::direction_v;
 
+  auto type() const -> const type_desc& {
+    static type_desc type;
+    return type;
+  }
```

Comment:
please don`t

Reply:
I need some way of getting the type of the slot, alternative suggestion are welcome

Reply:
this is also outdated, variable information is kept in a specified struct